.github/workflows/lint.yaml

```diff
@@ -1,49 +1,30 @@
 ---
 name: "Lint"
-on: # yamllint disable-line rule:truthy
+on:  # yamllint disable-line rule:truthy
   push:
     branches:
       - "main"
   pull_request:
     branches: ["*"]
 jobs:
   lint:
-    name: "Format & Lint"
+    # Uses pre-commit to run all checks
     runs-on: "ubuntu-latest"
-    strategy:
-      matrix:
-        python-version:
-          - "3.10"
-      fail-fast: false
     steps:
       - uses: "actions/checkout@v4"
-      - uses: "bewuethr/yamllint-action@v1.3.0"
+      - name: "Install Poetry"
+        uses: "snok/install-poetry@v1"
         with:
-          config-file: ".yamllint"
+          # This should come from pyproject.toml but poetry
+          # wasn't picking it up for whatever reason.
+          virtualenvs-in-project: true
+          virtualenvs-path: ".venv"
```

Comment:
I'm not super happy about this, but poetry doesn't otherwise seem to properly respect the pyproject.toml settings for this.